### DISCUSSION TITLE
如何使用排除選項

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [SiongSng/cap-countdown#42] [詢問] [answered]
https://github.com/SiongSng/cap-countdown/commit/1ec65b528b361f952caee967633599a88d2e4de6 增加了排除選項之功能，但是當我在選項上往左滑，只會往下一題前進

**Comment by SiongSng** [comment, marked as answer]
您好，本功能尚未正式上線到正式版本，無法使用是正常現象。

**Comment by SiongSng** [comment]
更新：本功能現已上線。
